nexus/compiler/Makefile

```diff
@@ -276,7 +277,7 @@ generate_code:
 	find . -name "*.bak" -type f -delete
 	@echo "Nexus Compiler: Moving files to output directory"
 	mkdir -p ${GENERATED_OUTPUT_DIRECTORY}/install-validator
-	cp -r ${GOPATH}/src/github.com/vmware-tanzu/graph-framework-for-microservices/install-validator/install.so ${GENERATED_OUTPUT_DIRECTORY}/install-validator/install.so || echo "Could not find install.so file, skipping"
+	# cp -r ${GOPATH}/src/github.com/vmware-tanzu/graph-framework-for-microservices/install-validator/install.so ${GENERATED_OUTPUT_DIRECTORY}/install-validator/install.so || echo "Could not find install.so file, skipping"
```

Comment:
is this ok?

Reply:
I have not found a place where that install.so binary is used.  Asking about it with the team that previously handled Nexus.